app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+
+class Planet:
+    def __init__(self, id, name, description, moons):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.moons = moons
+
+    def to_planet_dict(self):
+        return dict(
```

Comment:
👍 I like the `dict()` method for creating new dictionaries as well, since I can leave the quotes off the keys!